Yoast/wordpress-seo
# P1-13 Populate hidden input field in schema tab with data

## Context

* We want our data to persist after saving the page.

## Summary

This PR can be summarized in the following changelog entry:
* Populates the hidden input fields in the schema tab with data from the DB.
* Makes it possible for fields to be retrieved from the indexables table.

## Relevant technical choices:
* Added an option `indexable` that can be set to `true` to indicate whether that option should be fetched from the indexables table in the DB.

## Test instructions

This PR can be tested by following these steps:
Note: the fallback to the defaults cannot be tested conveniently until https://yoast.atlassian.net/browse/P1-3 is finished. 
1. Execute these steps for the following types: Posts, Custom Post Types and Pages
	1. Go to the edit page.
	1. Search for the following string `yoast_wpseo_schema_` in the browser's console or page source.
	1. On posts, this should result in two hidden input fields, on pages in one hidden input field.
	1. Check that the value is the same as the default for that type.
	1. Open the indexables table in your favorite DB editor (`wp_yoast_indexable`).
	1. Enter some values for `schema_article_type` and `schema_page_type` for the post / page ID (this is in `object_id`) that you are currently viewing. You can see the supported values in the options of the selects (in the pages source again).
	1. Reload the page and check that the values are visible in the page.
2. Make sure that there are no hidden input fields on taxonomy types
3. Make sure that there are no PHP errors on taxonomy pages

## Quality assurance


Fixes P1-13
